web_widget_timepicker/README.rst

```diff
@@ -0,0 +1,49 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+==============================
+Timepicker widget in form view
+==============================
+
+This module defines a timepicker widget, to be used with either char fields
+or (function) fields of type character. Use ``widget='timepicker'`` in your form view
```

Comment:
The motivation orginated out of the UI experience comparing to the datetimepicker widget for date/datetime input.
 
![timepicker_widget](https://cloud.githubusercontent.com/assets/19361409/15277004/08632b0a-1af8-11e6-9491-bc09d7ebd44d.PNG)

The existing float_time widget has limited UI support to assist the end user. The implemented time handling and persistance could be changed from Char to Float within the javascript code. Currently the Field Widget is based on AbstractField, so we are open for any changes.

PS: BTW and out of topic - Compliments for your great Oddo development essentials. book - it got me started.